common/src/main/scala/explore/components/Tile.scala

```diff
@@ -41,8 +39,7 @@ object Tile {
 
   type RenderInTitle = VdomNode ~=> VdomNode
 
-  // Explicitly never reuse as we are not considering the content
-  implicit val propsReuse: Reusability[Tile] = Reusability.never
+  implicit val propsReuse: Reusability[Tile] = Reusability.derive
```

Comment:
This is probably not taking the `render` parameter into account. I think we can add ` && Reusability.by(_.render)` at the end.

Reply:
I wonder if this is still the case in Scala 3, which has context functions. I'll give it a try before changing anything.

Reply:
Shouldn't `derive` take into account all fields?

Reply:
I'm not sure how the macro is implemented. There is a possibility it only takes a look at the first parameter group.